onboarding.md

```diff
@@ -0,0 +1,195 @@
+2. [Installing and Building](#InstallingandBuilding)
+3. [Running Locally](#RunningLocally)
+4. [Running Inside Docker](#RunningInsideDocker)
+5. [Testing](#Testing)
+6. [Common Errors/FAQs](#CommonErrorsFAQs)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+#  1. <a name='Prerequisites'></a>Prerequisites
+
+If you have followed the [general onboarding guide](https://github.com/mojaloop/mojaloop/blob/master/onboarding.md#mojaloop-onboarding), you should already have the following cli tools installed:
+
+* `brew` (macOS), [todo: windows package manager]
+* `curl`, `wget`
+* `docker` + `docker-compose`
+* `node`, `npm` and (optionally) `nvm`
+
+In addition to the above cli tools, you will need to install the following to build and run the `email-notifier`:
+
+
+###  1.1. <a name='macOS'></a>macOS
+```bash
+#none - you have everything you need!
+```
+
+###  1.2. <a name='Linux'></a>Linux
+
+[todo]
+
+###  1.3. <a name='Windows'></a>Windows
+
+[todo]
+
+
+##  2. <a name='InstallingandBuilding'></a>Installing and Building
+
+Firstly, clone your fork of the `email-notifier` onto your local machine:
+```bash
+git clone https://github.com/<your_username>/email-notifier.git
+```
+
+Then `cd` into the directory and install the node modules:
+```bash
+cd email-notifier
+npm install
+```
+
+> If you run into problems running `npm install`, make sure to check out the [Common Errors/FAQs](#CommonErrorsFAQs) below.
+
+##  3. <a name='RunningLocally'></a>Running Locally (with dependencies inside of docker)
+
+In this method, we will run core dependencies (`mongodb`, `kafka`) inside of docker containers, while running the `email-notifier` server on your local machine.
+
+> Alternatively, you can run the `email-notifier` inside of `docker-compose` with the rest of the dependencies to make the setup a little easier: [Running Inside Docker](#RunningInsideDocker).
+
+**1. Run the dependencies in `docker-compose`:**
+
+```bash
+# start the dependencies inside of docker
+docker-compose up mongo kafka
+
+```
+
+**3. Configure the default files and run the server**
+```bash
+# start the server
+npm run start
+```
+
+Upon running `npm run start`, your output should look similar to:
+
+```bash
+
+> email-notifier@6.2.0 start /<path_to>/email-notifier
+> WITH_SASL=0&&LD_LIBRARY_PATH=$PWD/node_modules/node-rdkafka/build/deps&& node app.js
+
+2019-06-03T07:23:27.850Z - info: Connection with database succeeded.
+2019-06-03T07:23:27.863Z - info: CreateHandle::connect - creating Consumer for topics: [topic-notification-event]
+2019-06-03T07:23:28.116Z - info: CreateHandle::connect - successful connected to topics: [topic-notification-event]
+healthcheck is listening on port 3080
+
+```
+
+##  4. <a name='RunningInsideDocker'></a>Running Inside Docker
+
+We use `docker-compose` to manage and run the `email-notifier` along with its dependencies with one command.
```

Comment:
Same as above. We need to write a `docker-compose.yml` file or write some instructions that asks the user to run the docker-compose from a different repo, e.g. `central-ledger`.